UnityProjects/MRTKDevTemplate/Assets/Scenes/ClippingInstancedExamples.unity

```diff
@@ -665,6 +665,243 @@ MeshRenderer:
   m_CorrespondingSourceObject: {fileID: 2300000, guid: 1104611d051d4e42a68391888b61b32e, type: 3}
   m_PrefabInstance: {fileID: 48993993}
   m_PrefabAsset: {fileID: 0}
+--- !u!21 &80620727
```

Comment:
@ms-RistoRK  please double check these changes.   Theis scene is changing more than just enabling the speech interactor.

Reply:
Double checked the scene, those extra changes in file were introduced by Unity Editor.  Just in case I repeated the process of copying .unity file from feature/XRI3 branch latest commit -> double-checked the new controllerless rig is used -> enabled MRTKSpeech rig component.
Resolving.